### PR TITLE
Errors are only excusable if they are rare, inadvertent and in good faith

### PR DESCRIPTION
**Pull request by pde**

### CODE REVIEW

File: dnt-policy-discussion-draft2.txt [hunk 1]
```diff
@@ -170,11 +170,12 @@ the following specific situations:
...
 5. ERRORS: 
 
-From time to time, there may be inadvertent errors by which user data is
-temporarily logged or retained in violation of this policy.  We will delete
-this data as soon practicable after we become aware of the error and take
-steps to ensure that it is deleted by any third-party who may have had access
-to the data.
+From time to time, there may be errors by which user data is temporarily
+logged or retained in violation of this policy.  If such errors are
+inadvertent, rare, and made in good faith, they does not constitute a breach
```
Comment (liliakai) [inline dnt-policy-discussion-draft2.txt]: > they s/does/do not

**Comment by quantumpacket** [comment]
> If such errors are inadvertent, rare, and made in good faith, they do not constitute a breach of this policy.

But what defines *rare*?

**Company A:** poorly trained admin breaches policy inadvertently and in good faith on avg 1/month.
**Company B:** poorly trained admin breaches policy inadvertently and in good faith on avg 1/yr.

I can say inadvertently breaching policy 1/month as rare, others can define it as 1/year. Clearly one has a worse of a track record than the other. Should they both be not in breach of policy when such errors occur?

I think some sort of baseline should be stated defining what constitutes as rare. That coupled with *good faith* can give a company a free pass as long as they play the *good faith* card as their violation rate has no defined limits. Sorry, if this is addressed elsewhere and I overlooked it.

**Comment by pde** [comment]
Good point @quantumpacket .  The problem with a precise numerical standard is that "rare" may actually be a function of organizational size.  A company with 100,000 employees and hundreds of products could easily make one error per month and argue somewhat plausibly that those errors were rare.  But for a small company that wouldn't be plausible.

My thinking had been that using a vague term like "rare" would be okay; it would only be in rare situations (such as an FTC investigation) that it would be deeply consequential whether a company had failed this standard or not, and in those situations lawyers could make arguments against each other about whether a particular frequency was indeed rare.

Having said that, if we can think of some somewhat flexible guidelines or standards to clarify rarity, it might be better to include them.

**Comment by pde** [comment]
Tentative conclusion from discussions: even though this standard is not a bright line, it is better to have it there than to not have it.

**Comment by pde** [comment]
(We talked about definitions of "rare" such as "rare means that users can have confidence that an error is not typically occurring", but it's not clear that the benefit is worth lengthening the document for)

**Pull request merged**